Microsoft.Toolkit.Uwp.UI.Controls.Core/ImageEx/ImageExBase.Source.cs

```diff
@@ -138,17 +138,7 @@ private async Task LoadImageAsync(Uri imageUri)
             {
                 if (IsCacheEnabled)
                 {
-                    switch (CachingStrategy)
-                    {
-                        case ImageExCachingStrategy.Custom when _isHttpSource:
-                            await SetHttpSourceCustomCached(imageUri);
-                            break;
-                        case ImageExCachingStrategy.Custom:
-                        case ImageExCachingStrategy.Internal:
-                        default:
-                            AttachSource(new BitmapImage(imageUri));
-                            break;
-                    }
+                    AttachSource(new BitmapImage(imageUri));
```

Comment:
Any thoughts if I should encapsulate this in a virtual method so that someone could inherit from ImageEx and then re-implement the custom caching method below on their own?

Reply:
I'm thinking that might be best. When we removed our usage of `System.Text.Json` we had a way for users to make there own serialization class to continue to use `System.Text.Json`.

It does not seem to cost us much and it makes the control more flexible.

Reply:
Yeah, exactly. I'll take a poke at that later today and come up with something. At least we know this is the path forward. 🙂

Reply:
Alright, I've reimplemented this to provide someone a way to due the old method within the confines of the control by creating their own subclass of it. Just need to test it and then will push it up here for final review.